### PR TITLE
Remove references only for SLE12 based deployer

### PR DESCRIPTION
**Pull request by dirkmueller**
The deployer and the CI job launcher have been switched
to Leap15, so we don't need to reference other internal
paths anymore that are only needed for SLE12. This reduces
the amount of red herrings a bit.

### CODE REVIEW

File: vars/deploy-on-openstack.yml [hunk 1]
```diff
@@ -18,13 +18,9 @@ deploy_on_openstack_sesnode_userdata: |
 deploy_on_openstack_repos_to_configure:
   SLE12SP3-product: http://provo-clouddata.cloud.suse.de/repos/x86_64/SLES12-SP3-Pool/
```
Comment (jgu17) [inline vars/deploy-on-openstack.yml]: is this repo accessible by the external tech preview users? this is not related to this PR :-)

Comment (dirkmueller) [reply vars/deploy-on-openstack.yml]: it is obviously not accessible by externals, but thats unrelated to this PR.

**Comment by evrardjp** [comment]
I don't see extra things to remove or to add for the 12sp3 deprecation in non SES.

**Comment by toabctl** [comment]
Great - our CI now just stops. That's why it is green...

**Comment by dirkmueller** [comment]
@toabctl whenever a PR is being merged it rebuilds all open PRs and that seems to confuse github. it looks like this needs fixing...
